include/swift/AST/DiagnosticsSema.def

```diff
@@ -4314,6 +4314,8 @@ ERROR(asynchandler_noescape_closure_parameter,none,
 ERROR(asynchandler_mutating,none,
       "'@asyncHandler' function cannot be 'mutating'",
       ())
+ERROR(asynchandler_removed,none,
```

Comment:
Was it truly removed if it was never added? Why not just diagnose it as an unknown attribute?

Reply:
People have code around that's played with `@asyncHandler`. I want this specific error so they know it was intentional. It'll become "unknown attribute" when we delete the code.